Maple2.Database/Storage/Game/GameStorage.User.cs

```diff
@@ -169,10 +170,10 @@ public partial class Request {
             return Context.TrySaveChanges();
         }
 
-        public (IList<KeyBind>? KeyBinds, IList<QuickSlot[]>? HotBars, List<SkillMacro>?, List<Wardrobe>?, IDictionary<StatAttribute, int>?, SkillBook?) LoadCharacterConfig(long characterId) {
+        public (IList<KeyBind>? KeyBinds, IList<QuickSlot[]>? HotBars, List<SkillMacro>?, List<Wardrobe>?, List<int>? FavoriteStickers, IDictionary<StatAttribute, int>?, SkillBook?) LoadCharacterConfig(long characterId) {
```

Comment:
`FavoriteStickers` is not nullable in db, so it can never be null here? If you want to make it nullable that's fine too but we should be consistent.